docs/deployment-on-arbitrary-cluster.md

```diff
@@ -49,6 +62,14 @@ extra work. I would recommend you to study the quickstart guide and modify it
 for your own case. Multus should soon include better documentation of the
 installation process.
 
+### OpenShift
```

Comment:
which openshift version suport multus? don't think this is officially out yet. please check

Reply:
It is only dev preview, not officially supported. I added a comment on that.